tests/integration_tests/db_engine_specs/presto_tests.py

```diff
@@ -33,42 +33,29 @@ class TestPrestoDbEngineSpec(TestDbEngineSpec):
     def test_get_datatype_presto(self):
         self.assertEqual("STRING", PrestoEngineSpec.get_datatype("string"))
 
-    def test_presto_get_view_names_return_empty_list(
```

Comment:
I'm not sure what the purpose of this function was, i.e., the logic can be inferred.